eth/tracers/live/supply.go

```diff
@@ -0,0 +1,247 @@
+package live
+
+import (
+	"encoding/json"
+	"log"
+	"math/big"
+	"path/filepath"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/eth/tracers/directory/live"
+	"github.com/ethereum/go-ethereum/params"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+func init() {
+	live.Directory.Register("supply", newSupply)
+}
+
+type SupplyInfo struct {
+	Delta       *big.Int `json:"delta"`
+	Reward      *big.Int `json:"reward"`
+	Withdrawals *big.Int `json:"withdrawals"`
+	Burn        *big.Int `json:"burn"`
+
+	// Block info
+	Number     uint64      `json:"blockNumber"`
+	Hash       common.Hash `json:"hash"`
+	ParentHash common.Hash `json:"parentHash"`
+}
+
+type supplyTxCallstack struct {
+	calls []supplyTxCallstack
+	burn  *big.Int
+}
+
+type Supply struct {
+	delta       SupplyInfo
+	txCallstack []supplyTxCallstack // Callstack for current transaction
+	logger      *log.Logger
+
+	// Check if genesis has been processed
+	hasGenesisProcessed bool
```

Comment:
This shouldn't be necessary anymore (I saw you merged in the latest).

Reply:
I will change it tomorrow. Thanks for the note.

Reply:
Done with https://github.com/s1na/go-ethereum/pull/21/commits/4dff9da4ca87b46e5621344fa720caabd0442df1